src/IntegrationUtils.ts

```diff
@@ -207,15 +256,18 @@ function createNewIntegration(integrationFileUri: vscode.Uri, devMode? : boolean
 		if (secret && secret.trim().length > 0) {
 			commandString += ` --secret=${secret}`;
 		}
-		if (resource && resource.trim().length > 0) {
+		if (resource !== undefined && resource.trim().length > 0) {
 			let resourceArray = resource.split(' ');
 			if (resourceArray && resourceArray.length > 0) {
 				resourceArray.forEach(res => {
 					commandString += ` --resource="${res}"`;
-				});
-			}			
+			});
+		}			
+		if (dependencyArray && dependencyArray.length > 0) {
+			dependencyArray.forEach(dependency => {
+				commandString += ` --dependency=${dependency}`;
+			});
 		}
-		commandString += ' "' + absoluteRoot + '"';
 		console.log(`commandString = ${commandString}`);
```

Comment:
logging required?

Reply:
Yes, I like having it there more for debugging purposes than anything. The user never sees it.